demand_acep/demand_acep.py

```diff
@@ -38,3 +51,11 @@ def extract_ppty(filename, meter_name):
             channel = meter_channel[n_name:]
 
     return meter, channel
+
+
+def data_resample(netcdf_df, sample_time='1T'):
+    """This function accepts a dataframe and downsamples it based on the sample time supplied"""
+    netcdf_resampled = netcdf_df.resample(sample_time).mean()
```

Comment:
@aidowu @reconjohn can we think of something other than mean for downsampling? We should have this discussion tomorrow.